src/lib/core/CHIPConfig.h

```diff
@@ -1611,6 +1611,15 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_SYNCHRONOUS_REPORTS_ENABLED 0
 #endif
 
+/**
+ * @def CHIP_CONFIG_SENDING_BATCH_COMMANDS_ENABLED
+ *
+ * @brief Device supports sending multiple batch commands in a single Invoke Request Message.
+ */
+#ifndef CHIP_CONFIG_SENDING_BATCH_COMMANDS_ENABLED
+#define CHIP_CONFIG_SENDING_BATCH_COMMANDS_ENABLED 0
+#endif
+
```

Comment:
I am not entirely convinced this config is needed. I added it in case we did want deviced to opt into supporting sending batch commands

Reply:
Keeping this define for now, we can decide later to remove it fairly easily.